infra.lvm_tools/shrink_lv/meta/runtime.yml

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
Since this file is all commented out, you can delete it altogether

Reply:
ACK. I was not sure if there was anything I'd have to keep.